swarm/pss/pss_test.go

```diff
@@ -457,6 +457,76 @@ func TestKeys(t *testing.T) {
 	}
 }
 
+func TestGetPeer(t *testing.T) {
```

Comment:
There is no more GetPeer, so TestGetPeer is not the best name.... as I see it gets the topics and the address too... maybe `TestGetPeerData`?